apps/nextjs-website/src/components/organisms/Ecosystem/Ecosystem.tsx

```diff
@@ -0,0 +1,37 @@
+'use client';
+import React from 'react';
+import { Box, Typography, useTheme } from '@mui/material';
+import TabComponent, {
+  TabItem,
+} from '@/components/atoms/TabComponent/TabComponent';
+
+type EcosystemProps = {
+  title?: string;
+  backgroundColor?: string;
+  items: ReadonlyArray<TabItem>;
+};
+
+const Ecosystem = ({ title, backgroundColor, items }: EcosystemProps) => {
+  const theme = useTheme();
+  return (
+    <Box
```

Comment:
I think you should fix some distances:
![image](https://github.com/pagopa/developer-portal/assets/39835990/7f3f5d77-1b48-4db1-a3dc-7bf50028d726)
![image](https://github.com/pagopa/developer-portal/assets/39835990/9ad0b958-2daa-4ad2-84f7-1f390c5ccabd)
![image](https://github.com/pagopa/developer-portal/assets/39835990/ff55a480-de4f-4de7-a415-c7b0a2885dea)
![image](https://github.com/pagopa/developer-portal/assets/39835990/c517f061-a968-4aa0-bbb3-ae236bc762a1)